.github/workflows/regression.yml

```
@@ -0,0 +1,84 @@
+on:
+  pull_request:
+    branches: [ "master" ]
+
+name: "Regression"
+
+jobs:
+  compare-results:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout PR branch
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 2
+
+    - uses: dtolnay/rust-toolchain@stable
+
+    - name: Download LAZ file
+      env:
+        REGRESSION_LAZ_FILE_URL: ${{ secrets.REGRESSION_LAZ_FILE_URL }}
+      run: curl -L "$REGRESSION_LAZ_FILE_URL" -o test_file.laz
```

> """With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository.""" Let's use a variable instead. preference for name like KP_TEST_LAZ_URL

So we could do something like:

```suggestion
      env:
        KP_TEST_LAZ_URL: ${{ vars.KP_TEST_LAZ_URL }}
      run: curl -L "$KP_TEST_LAZ_URL" -o test_file.laz
```